packages/inspector/src/Tree.tsx

```diff
@@ -395,12 +407,21 @@ export default function Tree({
     }
   }
 
+  const [expanded, setExpanded] = useState(getExpanded(path));
```

Comment:
This is duplicating the state down throughout the tree, which can cause problems because you have to keep things in sync. You shouldn't need to use `useState` here at all; you can rely on `getExpanded` and `onExpand`.

Reply:
Yep, removed it in a following commit :)